packages/fxa-auth-server/lib/senders/emails/partials/button.ts

```diff
@@ -1,9 +1,8 @@
 export const button = `
-  <mj-section>
-    <mj-column>
-      <mj-button>
-        <%= buttonText %>
-      </mj-button>
-    </mj-column>
-  </mj-section>
+<mj-include path="./lib/senders/emails/css/button.css" type="css" css-inline="inline" />
+    <mj-section>
+      <mj-column>
+        <mj-button css-class="p-button"><a href="www.google.com"><%= buttonText %></a></mj-button>
```

Comment:
Is this intentionally `www.google.com`? Just temporary for development?

Reply:
Yeah just wanted to test if it's working in a email client, but I'll remove it in the next commit